js/views/search.vue

```diff
@@ -37,29 +37,38 @@ export default {
         discussions: require('components/discussions/list.vue'),
         Layout
     },
+    computed: {
+        title() {
+            return [
+                this._('Search in your data'),
+                this.$route.query.q
+            ].join(': ');
```

Comment:
You need a translation here for the `:` (takes a non-breaking space before in French).

BTW, I don't why you join and not use a simple concatenation or maybe a template string?